### DISCUSSION TITLE
Failed to load /config.json  esp32 d1 mini Lolin D32 pro

### DISCUSSION THREAD

**Discussion by Grey-Lancaster** [HASwitchPlate/openHASP#620] [Hardware]
I have tried platformio and the web installer.  In platformio it looks like it is uploading the data files but when it boots it does not load config.json and the access point does not have an ip?

Changing baud rate to 921600
Changed.
Configuring flash size...
Flash will be erased from 0x00001000 to 0x00004fff...
Flash will be erased from 0x00008000 to 0x00008fff...
Flash will be erased from 0x0000e000 to 0x0000ffff...
Flash will be erased from 0x00010000 to 0x001b7fff...
Compressed 15920 bytes to 11195...
Writing at 0x00001000... (100 %)
Wrote 15920 bytes (11195 compressed) at 0x00001000 in 0.2 seconds (effective 539.7 kbit/s)...
Hash of data verified.
Compressed 3072 bytes to 153...
Writing at 0x00008000... (100 %)
Wrote 3072 bytes (153 compressed) at 0x00008000 in 0.0 seconds (effective 1068.3 kbit/s)...
Hash of data verified.
Compressed 8192 bytes to 47...
Writing at 0x0000e000... (100 %)
Wrote 8192 bytes (47 compressed) at 0x0000e000 in 0.0 seconds (effective 1725.0 kbit/s)...
Hash of data verified.
Compressed 1733008 bytes to 1182520...
Writing at 0x00010000... (1 %)
Writing at 0x00016ad1... (2 %)

logs
        open____ _____ _____ _____
          |  |  |  _  |   __|  _  |
          |     |     |__   |   __|
          |__|__|__|__|_____|__|
        Home Automation Switch Plate
        Open Hardware edition v0.7.0-rc9

[       0.041][110580/183252 39][    0/    0  0] DBUG: Started @ 115200 bps
#[       0.050][110580/183252 39][    0/    0  0] DBUG: Environment: lolin-d32-pro_ili9341
#[       0.060][110580/183252 39][    0/    0  0] UART: Started
#[       0.068][110580/183252 39][    0/    0  0] UART: Client login from serial
#[       0.077][110580/183252 39][    0/    0  0] CONF: Failed to load /config.json
Prompt > [       0.087][110580/183252 39][    0/    0  0] CONF: Loading EEPROM
Prompt > [       0.096][110580/183252 39][    0/    0  0] CONF: Loaded EEPROM
Prompt > [       0.105][110580/183316 39][    0/    0  0] DBUG: Loading debug settings
Prompt > [       0.115][110580/183316 39][    0/    0  0] GPIO: Loading GUI settings
Prompt > [       0.125][110580/183316 39][    0/    0  0] HASP: Loading HASP settings
Prompt > [       0.135][110580/183316 39][    0/    0  0] WIFI: Loading WiFi settings
Prompt > [       0.145][110580/183316 39][    0/    0  0] ----: Failed to open partition
Prompt > [       0.155][110580/183316 39][    0/    0  0] WG  : Loading WireGuard settings
Prompt > [       0.166][110580/183316 39][    0/    0  0] ----: Failed to open partition
Prompt > [       0.176][110580/183316 39][    0/    0  0] MQTT: Loading MQTT settings
Prompt > [       0.186][110580/183316 39][    0/    0  0] ----: Failed to open partition
Prompt > [       0.196][110580/183316 39][    0/    0  0] TELN: Loading Telnet settings
Prompt > [       0.206][110580/183316 39][    0/    0  0] MDNS: Loading MDNS settings
Prompt > [       0.216][110580/183316 39][    0/    0  0] HTTP: Loading HTTP settings
Prompt > [       0.225][110580/183316 39][    0/    0  0] ----: Failed to open partition
Prompt > [       0.235][110580/183316 39][    0/    0  0] GPIO: Loading GPIO settings
Prompt > [       0.245][110580/183316 39][    0/    0  0] CONF: Settings loaded
Prompt > [       0.258][110580/170928 35][    0/    0  0] ----: Failed to open partition
Prompt > [       0.268][110580/170928 35][    0/    0  0] ----: Failed to open partition
Prompt > [       0.279][110580/170688 35][    0/    0  0] MSGR: Starting...
Prompt > [       0.288][110580/170688 35][    0/    0  0] MSGR: Started
Prompt > [       0.296][110580/170688 35][    0/    0  0] TFT : Starting...
Prompt > [       0.636][110580/170532 35][    0/    0  0] TFT : Started
Prompt > [       0.645][110580/170532 35][    0/    0  0] LVGL: Starting...
Prompt > [       0.661][77812/137548 43][63044/63232  1] LVGL: LV_FS_PC ready
Prompt > [       0.671][77812/137548 43][63044/63224  1] LVGL: /littlefs/config.json
Prompt > [       0.680][77812/137548 43][63044/63232  1] HASP: TEST Opening config.json from FS failed 12
Prompt > [       0.694][77812/111324 30][62308/62356  1] GUI : Initialize Cursor
Prompt > [       0.704][77812/111324 30][62308/62356  1] ----: Failed to open partition
Prompt > [       0.718][77812/111224 30][61976/61992  1] LVGL: Started
Prompt > [       0.727][77812/111224 30][61976/61992  1] WIFI: Temporary Access Point HASP-90f50c password: haspadmin
Prompt > [       0.739][77812/111160 30][61976/61992  1] WIFI: AP IP address : ::
Prompt > [       0.751][77812/110520 29][61976/61992  1] HTTP: Started @ http://::

**Comment by fvanroie** [comment]
`Failed to open partition` is not normal here. Please erase the full flash first and try again...

**Reply by Grey-Lancaster** [reply]
yea, I have tried that.  I dunno, I thought somehow "re-using" these chips caused me problems.  Like it would work for openHasp then if I tried esphome it would fail, and then I could not get openhasp to work again. But I tried a new chip and still problems.
I guess just a bunch of bad chips.

upload filesystem image in platformio fails md5 chksum
Wrote 12517376 bytes (286446 compressed) at 0x00410000 in 29.7 seconds (effective 3370.0 kbit/s)...
File  md5: 27d0cf30302b7dff3fd365991e80308c
Flash md5: 4ed38ee8aab9b0d19e71cd4a414462f5
MD5 of 0xFF is ff6d0a62ff08286feecbc8cb857a4130

**Comment by Grey-Lancaster** [comment]
@fvanroie Thanks for looking and I apologize for wasting your time. I am new to openHASP and get confused sometimes
I thought the Lolin was for the Lolin 2.4 tft but that is the chip
I was trying to install 16mb to an esp32 d1 mini
When I selected the right download boom badada boom !  Works great
